feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/productTypeConversions.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.google
+
+import com.android.billingclient.api.BillingClient
+import com.revenuecat.purchases.ProductType
+
+fun String?.toProductType(): ProductType {
```

Comment:
wondering whether it's worth renaming to `RCProductType` now that both Amazon and Google have a `ProductType`

Reply:
Oh I actually made that comment in another file. I think we should consider renaming yes. What is iOS using? We should at least rename the extension function

Reply:
on iOS it looks like it's `ProductType` (though still `RCProductType` in objc